app/policies/gem_name_reservation_policy.rb

```diff
@@ -0,0 +1,27 @@
+class GemNameReservationPolicy < ApplicationPolicy
+  class Scope < Scope
+    def resolve
+      scope.all
+    end
+  end
+
+  def avo_index?
+    true
+  end
+
+  def avo_show?
+    true
+  end
+
+  def avo_create?
```

Comment:
Create and destroy should be limited to rubygems.org admins

Reply:
Thanks, I have updated this. I was looking into other policies to inspire. Is there currently any difference in this behaviour?